third_party/chromium/media/base/starboard_utils.cc

```diff
@@ -396,6 +399,24 @@ SbMediaColorMetadata MediaToSbMediaColorMetadata(
 
   return sb_media_color_metadata;
 }
+
+int GetSbMediaVideoBufferBudgetMaximum() {
+  LOG(INFO) << "Checking for maximum video buffer budget.";
+  auto budget = SbMediaGetVideoBufferBudget(SbMediaVideoCodec::kSbMediaVideoCodecH264, 1920, 1080, 8);
```

Comment:
The line is longer than 80 chars, consider formatting.

Reply:
removed.